app/upgrades/v4/upgrade.go

```diff
@@ -78,6 +78,14 @@ func CreateUpgradeHandler(
 			return nil, err
 		}
 
+		if err := migrateRollappRegisteredDenoms(ctx, keepers.RollappKeeper); err != nil {
+			return nil, err
+		}
+
+		if err := migrateDymnsParams(ctx, keepers.DymNSKeeper); err != nil {
```

Comment:
I don't think we need to migrate dymns params. just add the dymns to the new modules list in the `Added` of the create upgrade handler

Reply:
done! plus also added sponsorship and iro modules

Reply:
nice. I think we're missing `grouptypes`